.ci/magic-modules/pyutils/strutils.py

```diff
@@ -39,62 +39,65 @@ def parse_github_url(gh_url):
     return (repo, int(prnum))
   return None
 
-def get_release_note(body):
-  """Parse release note block from a given text block.
+def get_release_notes(body):
+  """Parse release note blocks from a given text block.
 
-  Finds the first markdown code block with a "releasenote" language class.
+  Each code-block with a "release-note:..." language class.
   Example:
-    ```releasenote
```

Comment:
It looks like it, but this code will still match '''releasenote, right, without the colon?

Reply:
nope! it'll match "release-note: ..." - is that not what we want?

Reply:
Is it possible to have it match both? For us it's immaterial, it just struck me as nice if it matched the tool. ::shrug:: not a big deal